src/main/groovy/de/dkfz/roddy/execution/jobs/cluster/lsf/rest/LSFRestJobManager.groovy

```diff
@@ -248,22 +261,14 @@ class LSFRestJobManager extends BatchEuphoriaJobManagerAdapter {
      * @return part of parameter area
      */
     private String prepareToolScript(BEJob job, String boundary) {
-        String toolScript
-        if (job.getToolScript() != null && job.getToolScript().length() > 0) {
-            toolScript = job.getToolScript()
-        } else {
-            if (job.getTool() != null) toolScript = job.getTool().getAbsolutePath()
-        }
-        if (toolScript) {
-            return ["--${boundary}",
-                    "Content-Disposition: form-data; name=\"COMMAND\"",
-                    "Content-Type: application/xml; charset=UTF-8",
-                    "Content-Transfer-Encoding: 8bit",
-                    "Accept-Language:en-en\r\n",
-                    "<AppParam><id>COMMANDTORUN</id><value>${toolScript}</value><type></type></AppParam>\r\n"].join("\r\n")
-        } else {
-            return ""
-        }
+
+        return ["--${boundary}",
+                "Content-Disposition: form-data; name=\"COMMAND\"",
+                "Content-Type: application/xml; charset=UTF-8",
```

Comment:
Why is the encoding different to the one above? Could we extract some clear functions? Or avoid code dup?

Reply:
I refactored the methods a bit and removed headers that don't seem to be necessary.